phys/module_surface_driver.F

```diff
@@ -3264,7 +3264,7 @@ SUBROUTINE surface_driver(                                         &
                IF(PRESENT(ACGRDFLX))ACGRDFLX(I,J)=ACGRDFLX(I,J) + GRDFLX(I,J)*DT
 
                ! Check that SFCDIAGS can declare these as intent(out)
-               T2(I,J)  = -1.E36
+               T(I,J)  = -1.E36
```

Comment:
Negin,
Does this exist?
There is a 3d t_phy.

Reply:
@davegill Thanks for pointing this out. This should have been T2 and unchanged. I've fixed in the module.